integrations/sqlite.ini

```diff
@@ -58,6 +58,7 @@ PROVIDER = file
 MODE                 = test,file
 ROOT_PATH            = sqlite-log
 REDIRECT_MACARON_LOG = true
+DISABLE_ROUTER_LOG = true
```

Comment:
I think it is one left over of local testing. Can you revert this change ?

Reply:
```suggestion
```

Reply:
Done